### PR TITLE
Add support for retrieving a prisoner photo from NOMIS

### PR DESCRIPTION
**Pull request by ibrechin**
This photo is to be used on the prisoner profile page.

### CODE REVIEW

**Comment by ibrechin** [comment]
@vickymg this is the backend work for the photo. On the page you should be able to check if the API integration is available with `{% if nomis_api_available %}` and if the photo is present with `{% if photo %}` (`photo` is b64 encoded image data so should be able to use it with just ``.

File: mtp_noms_ops/templates/security/prisoners-detail.html [hunk 1]
```diff
@@ -2,20 +2,32 @@
...
-      {{ view.title }}
       {% if prisoner %}
+        {% if nomis_api_available %}
+          <div class="mtp-prisoner-image">
+            {% if photo %}
+              <img src="{{ photo }}" alt="{% trans 'Photo of the prisoner' %}"/>
+            {% else %}
+              <img src="{% static 'images/placeholder-image.png' %}"
+              srcset="{% static 'images/placeholder-image@2x.png' %} 2x"
+              alt="{% trans 'Placeholder image for prisoner photo' %}" />
```
Comment (ushkarev) [inline mtp_noms_ops/templates/security/prisoners-detail.html]: this wouldn't be a placeholder so screen readers would say the wrong thing

File: mtp_noms_ops/templates/security/prisoners-detail.html [hunk 2]
```diff
@@ -2,20 +2,32 @@
...
   <header>
     {% language_switch %}
-    {% include 'security/includes/save-search.html' %}
     <h1 class="heading-xlarge">
-      {{ view.title }}
       {% if prisoner %}
+        {% if nomis_api_available %}
+          <div class="mtp-prisoner-image">
+            {% if photo %}
+              <img src="{{ photo }}" alt="{% trans 'Photo of the prisoner' %}"/>
```
Comment (ushkarev) [inline mtp_noms_ops/templates/security/prisoners-detail.html]: should be `data:image/jpeg;base64,{{ photo }}` i think. but also depends on how NOMIS encodes the image – sometimes base64 encoded files include line breaks that we’d have to strip out.

**Comment by ushkarev** [comment]
oh and a rebase is needed now

**Pull request merged**